tests/integration/helpers.py

```diff
@@ -362,21 +362,23 @@ async def get_unit_hostname(ops_test: OpsTest, unit_id: int, app: str) -> str:
     return hostname.strip()
 
 
-def get_raw_application(ops_test: OpsTest, app: str) -> Dict[str, Any]:
+async def get_raw_application(ops_test: OpsTest, app: str) -> Dict[str, Any]:
     """Get raw application details."""
-    return json.loads(
-        subprocess.check_output(
```

Comment:
Nothing for you to change here, just some context

I believe the idea here of using `subprocess` instead of doing something like `ops.model.application.status` or something like that is that sometimes `ops_test` isn't the most reliable when it comes to statuses

here you use `ops_test.juju` to retrieve the status (doing essentially what we were doing before), which should be fine, even if it does rely on `ops_test`.